provider/resources.go

```diff
@@ -216,7 +214,7 @@ func Provider() tfbridge.ProviderInfo {
 				"@types/node": "^10.0.0", // so we can access strongly typed node definitions.
 				"@types/mime": "^2.0.0",
 			},
-			PackageName: "@jaxxstorm/pulumi-scaleway",
+			PackageName: "@pulumiverse/pulumi-scaleway",
```

Comment:
@jaxxstorm package name should be `@pulumiverse/scaleway`

Reply:
Package name must have the `pulumi-` part dropped to be similar to all the other generated Pulumi JS/TS packages.